src/cpp/shared/combat/army.cpp

```diff
@@ -5,6 +5,9 @@
 #include "scripting/callback.h"
 #include "sound/sound.h"
 #include "spell/spells.h"
+#include "expansions.h"
+
+extern ironfist_expansions ironfist_exp;
```

Comment:
I think we're keeping the original HoMM II naming convention for globals: gIronfistExp,